platform/src/main/java/org/hillview/storage/GenericLogs.java

```diff
@@ -80,25 +68,28 @@ void parse(String line, String[] output) {
         public ITable load() {
             try (BufferedReader reader = new BufferedReader(
                     this.getFileReader())) {
```

Comment:
@mbudiu-vmw Please add the removed code to handle the empty file.
Else will face the NullPointerException.
0ccb64ea-9e84-4250-b0c1-b49d43335e32.loadTable() java.lang.Exception: java.lang.NullPointerException 
at org.hillview.dataset.LocalDataSet.lambda$map$1(LocalDataSet.java:118) 
Caused by: java.lang.NullPointerException

Reply:
I will fix this, but if a log file is empty you will have other problems when you load data from multiple machines. The schema will not match between the various machines because empty files will have no columns. What we need to do is to create a schema even if there are no lines - the schema will need to be created directly from the pattern instead of being created from the first row.